README.md

```diff
@@ -33,6 +33,10 @@ The **Meesterproef** is the final client-project of the half year minor programm
 
 ### Tech stack
 
+## Prerequisites
+
+- Redis database for session storage
```

Comment:
Wat is redis? Wat is session storage? Sorry, het is kut, maar kan wel iets uitgebreider. Je moet zo denken: stel dat iemand (die nieuw is) dit project gebruikt, moet deze het kunnen begrepen zonder met ons contact op te nemen. Desnoods link je naar de Docker docs ofzo. Maar ik zou (als ik nieuw was in het project) niet weten waarom ik dit nodig zou hebben.

Reply:
Uitleg van Redis en session storage moet in de design rationale, in my opinion. Kan wel doorlinken naar Redis. Uiteindelijk kan je hetzelfde zeggen over Node.js of mongoDB ...

Reply:
Correct. Ik vind dat ook Node en MongoDB toegelicht moeten worden. Daar is waar heel de README voor bedoeld imo.  Wat ik zei: je kunt ook gewoon linken naar de docs.

```markdown

For this project we use Node (link) as back-end and mongoDB (link) as database.

## Database Installation

### 1. ...

### 2. ...

etc.


```

Maar ik vind `Redis database for sessions storage` nogal een korte omschrijving die voor de rest niets zegt.

Reply:
Dan doe ik session storage uitleggen in de design rationale en dan linken vanaf readme. Mensen die niet weten wat het is kunnen dan daar naar door klikken :) 

Misschien kunnen we het zelfde doen voor database en node.js?

Reply:
Jaa prima. Gaat meer om dat de basis gewoon in de README staat. Als ze meer willen weten, dan klikken ze maar verder.